### ISSUE TITLE
Dark Mode on Facebook Messenger no longer working

### ISSUE THREAD

**Issue by Julian88Tex** [getferdi/ferdi#1077]
**Bug description:**
Dark mode is no longer working on Facebook Messenger. I've tried restarting Ferdi, disabling and enabling again, and adding and removing service with no luck.

**Steps to reproduce:**

1. Add Facebook Messenger as a service
2. Verify Dark Mode is enabled in Ferdi settings
3. Refresh Facebook

**Expected behavior**
Facebook Messenger is in dark mode

**Screenshots**
![Screen Shot 2020-11-18 at 12 48 05 PM](https://user-images.githubusercontent.com/459099/99586295-63be8780-299c-11eb-8391-382dd6a426b5.png)
<img width="638" alt="Screen Shot 2020-11-18 at 12 45 56 PM" src="https://user-images.githubusercontent.com/459099/99586307-66b97800-299c-11eb-86ce-690f7067d377.png">

**Environment:**

- Operating System: macOS Catalina 10.15.7
- Ferdi Version: 5.5.0
- Server: Ferdi

- Debug information:
[ferdidebugfacebooknotdark.txt](https://github.com/getferdi/ferdi/files/5562695/ferdidebugfacebooknotdark.txt)

**Comment by i-need-to-tell-you-something** [comment]
By "disabling and enabling" do you mean the service itself or the dark mode? 

For me the FB dark mode often stops working on Linux, but toggling the dark mode off and on fixes it. I don't remember if I restart the app after that as well or not. 

Also one possible detail to pay attention to might be whether Ferdi is hidden or actually closed when you're trying to restart it. You might want to check your System Monitor / Task Manager (sorry, don't remember how it was called in macOS).

**Comment by akimonlet** [comment]
Same here, it was working since weeks with a custom css and yesterday it just stopped.

**Comment by Julian88Tex** [comment]
@i-need-to-tell-you-something thanks for the recommendation. Unfortunately, I just gave this a try but no luck :(. Instagram dark mode stopped working yesterday so in total these 3 services are not working for me in dark mode:

- LinkedIn
- Facebook Messener
- Instagram

**Comment by jpanaro** [comment]
Just to chime In, messenger dark mode does not work for me as well so @Julian88Tex is not the only one with the issue.

**Comment by aalvarado** [comment]
seems to me that the web app has updated the class names and the one's in the dark css file no longer match the one's in the web app. I was able to solve this issue by editing the dark css file for messenger and the developer console. The comments in the darkcss file help guide you were to add the css classes

**Comment by Julian88Tex** [comment]
@aalvarado oh that sounds awesome, would you mind adding your changes to a PR or gist?

**Comment by aalvarado** [comment]
@Julian88Tex 
I spent some time trying to find where the dark css file was but wasn't able to find it. if you can provide a location, I can add the missing classes

**Comment by Julian88Tex** [comment]
@aalvarado Thanks for trying that, I did the same but couldn't find one either. I haven't looked deeply into dark mode references but I suspect it is something that is generated as you create a service and/or turn on dark mode for that service.

That being said, I also remembered this repo for darkmode.css files for various services. I replaced my local file and it seems to be working well. If you have a second, feel free to take a look and recommend any improvements if you happen to see them: https://github.com/ducfilan/Dark-mode-Franz-Ferdi/blob/master/messenger/darkmode.css

Appreciate the help either way!

**Comment by snorkem** [comment]
This issue is still relevant for me. Dark mode icons do not look good (too dark, not enough contrast) on facebook messenger.

**Comment by i-need-to-tell-you-something** [comment]
> This issue is still relevant for me. Dark mode icons do not look good (too dark, not enough contrast) on facebook messenger.

I agree, but I would argue that that is a separate issue from this one and that this one can be closed.

**Comment by vraravam** [comment]
> seems to me that the web app has updated the class names and the one's in the dark css file no longer match the one's in the web app. I was able to solve this issue by editing the dark css file for messenger with the help of the developer console. The comments in the darkcss file help guide you were to add the css classes

@aalvarado - the recipes are in this repo: http://github.com/getferdi/recipes - if that is too cumbersome, please attach your changes into this discussion, and I can merge it into the appropriate repo.
